kura/org.eclipse.kura.ble.provider/src/main/java/org/eclipse/kura/internal/ble/BluetoothLeAdapterImpl.java

```diff
@@ -144,6 +144,14 @@ public boolean isDiscovering() {
         return uuidList.toArray(uuids);
     }
 
+    public void startDiscovery() throws KuraBluetoothDiscoveryException {
+        try {
+            this.adapter.startDiscovery();
+        } catch (BluetoothException e) {
+            throw new KuraBluetoothDiscoveryException(e, "Start discovery failed");
+        }
+    }
+
```

Comment:
I think that we can add the signature of this method as API in the `BluetoothLeAdapter`.

Reply:
This makes sense, I just wasn't sure if you wanted to expose this functionality in the API directly given the results of starting a discovery would not be consumable through existing API methods.

Reply:
Yes, I understand your concern... but I prefer to have it in the API (along with the `stopDiscovery`..)